source/common/upstream/conn_pool_map_impl.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "common/upstream/conn_pool_map.h"
+
+namespace {}
```

Comment:
?

Reply:
Left it in after some intermediate refactoring. Whoops.